### DISCUSSION TITLE
effect on energy calculations

### DISCUSSION THREAD

**Discussion by colesadam** [hif2k1/battery_sim#85] [Q&A] [answered]
Hi, this looks exactly what I need to try and work out if a battery would be worth it for me but if I integrate it with the energy dashboard does that mean the figures it outputs for solar, cost etc will stop being correct? Is there a way of calculating usage that doesn't affect the normal true energy stats?

**Comment by hif2k1** [comment]
If you integrate it with the energy dashboard that will affect your
readings of energy used and exported in the dashboard history yes. It won't
alter the underlying sensors like your meter reading sensor and it
shouldn't affect readings of solar generated. I never really imagined
integrating it into the energy dashboard because it doesn't actually add
anything except make it look like you have a battery you don't actually
have, but people requested it. The Battery Sim will work perfectly well on
it's own without integrating it.

**Reply by colesadam** [reply]
Thanks for your reply. What do you find is the best way of visualising the data, specifically if over periods of time I want to understand whether the amount of savings is worth the cost of ownership?

**Comment by hif2k1** [comment, marked as answer]
I think the best way is to set up two or three different batteries that you
might be considering e.g. a small cheaper one and a big one and then leave
it running for several months. Ideally you want data from summer and winter
or over a whole year if possible. Then you can see how much energy you save
(Total energy saved sensor) or oney saved if you have the tariffs set up
and work out what the savings and payback period would be and whether it's
worth going for a bigger battery or if the extra savings would be marginal.
